server/src/clients/google.py

```diff
@@ -92,3 +92,38 @@ def get_user_info(
             parsed_user_info[field] = user_info.get(field, "")
 
         return (None, parsed_user_info)
+
+
+@error_bounded(
+    (AppError(ErrorCode.GOOGLE_API_ERROR, "Google api error - get user contacts"), None)
+)
+def get_people_connections_list(token: str,) -> Tuple[Optional[AppError], List[object]]:
+    """
+    Request list of user's contacts from Google People API.
+
+    Returns a list of Google API Person objects representing all
+    of the user's contacts
+    """
+    creds = Credentials(token) if token else None
+    with build("people", "v1", credentials=creds) as people_service:
+        pagination_finished = False
+        next_page_token = ""
+        people_list_complete = []
+        # paginate through contacts and add to list
+        while not pagination_finished:
+            people_response = (
```

Comment:
Is is possible for this to return an error?

Reply:
Yeah it's possible for a 410 sync token error or a 429 quota error; is there anything I need to do for that or does the errorBounded thing handle that?